pkg/errors/errors.go

```diff
@@ -36,6 +36,11 @@ type CommandError struct {
 	Reason error
 }
 
+// Error returns the error message
+func (e UserError) Is(target error) bool {
```

Comment:
I'm sorry, this is not correct and I didn't catch it previously. This is still just checking strings. There should be no string logic in error comparisons. It's too brittle and error strings can not be used for logic. 

We must either use a type assertion or the `==` operator to check and exact match using the type system and not string logic.